lib/modules.nix

```diff
@@ -681,6 +686,8 @@ rec {
       map (mapAttrs (n: v: mkIf cfg.condition v)) (pushDownProperties cfg.content)
     else if cfg._type or "" == "override" then
       map (mapAttrs (n: v: mkOverride cfg.priority v)) (pushDownProperties cfg.content)
+    else if cfg._type or "" == "postmerge" then
+      map (mapAttrs (n: v: mkPostMerge cfg.f v)) (pushDownProperties cfg.content)
```

Comment:
that does not look right. pushing down `postmerge` is a guarantee for indecipherable type errors at eval time when a non-leaf option is `mkPostMerge`d.

Reply:
It just makes sure that the content argument is in the same form as all the other definitions

Reply:
ah, i mislinked where the postmerge nodes go. this looks fine in fact.